### PR TITLE
add ERAPraktikum and GED

### PR DESCRIPTION
**Pull request by Geosearchef**

### CODE REVIEW

File: redirect/routes.php [hunk 1]
```diff
@@ -171,7 +171,15 @@ class Route {
 	'pgm' => [
 	      'description' => "Probabilistic Graphical Models in Computer Vision",
 	      'target' => 'http://vision.in.tum.de/teaching/ss2017/pgmcv'
-	]
+	],
+        'eraprak' => [
```
Comment (Geosearchef) [inline redirect/routes.php]: anyone got a better abbreviation? there is no era this term but I'd still avoid changing between normal lecture and the "praktikum" each term

**Comment by mammuth** [comment]
Hm, maybe just use erapraktikum.tum.sexy?
Isn't short anymore but more memorable than eraprak in my opinion. 

That chair has a valid TLS certificate I think, let's redirect to https if that's correct :)

**Comment by Geosearchef** [comment]
i agree, changed, https added

**Comment by mammuth** [comment]
Great, thanks!

**Pull request merged**